stl/inc/deque

```diff
@@ -1577,6 +1689,12 @@ template <class _Iter, class _Alloc = allocator<_Iter_value_t<_Iter>>,
 deque(_Iter, _Iter, _Alloc = _Alloc()) -> deque<_Iter_value_t<_Iter>, _Alloc>;
 #endif // _HAS_CXX17
 
+#ifdef __cpp_lib_containers_ranges
+template <_RANGES input_range _Rng, class _Alloc = allocator<_RANGES range_value_t<_Rng>>,
+    enable_if_t<_Is_allocator<_Alloc>::value, int> = 0>

```

Comment:
Dont we have a inline variable template for that?

Reply:
Funny - Stephan asked the same thing during Open Code Review. No we don't.

Reply:
Should we have an iniline variable for that?

Reply:
In this PR? No. In some other PR? I suppose it wouldn't hurt. The Standard only uses "qualifies as an allocator" to constrain deduction guides, so it doesn't have a lot of uses; I'm honestly indifferent.